### DISCUSSION TITLE
Lost strokes on RX800

### DISCUSSION THREAD

**Discussion by fabfire** [JaapvanEkris/openrowingmonitor#67] [Rower Profile Development]
Hello,

I've made a training session with my setup : ORM and Norditrack RX800.
I notice that sometimes, during the session, the values of strokes count, SPM, watts, .... are missing. 
Not at the beginning, after a few strokes, most of the values are missing. But the total distance and time is still increasing.

Files are attached, is this session I made 179 strokes but ORM reports 147.
In the elapsed time, we see a stroke each 3 seconds approx. 
But between stroke 80 and 81, there is 6 sec (one stroke is missing) and between stroke 84 and 85, there is 26 seconds but I kept the same cadence in reality.  

<img width="1255" alt="image" src="https://github.com/user-attachments/assets/ad21137c-f89f-4010-98c3-d9b4cbf30ab8" />

I also look at raw data and curves seems correct (but I'm not an expert).  

<img width="1170" alt="image" src="https://github.com/user-attachments/assets/2619b321-3970-4c7c-8f92-e2af72a2fd89" />

[2024-12-13_15-23-25_raw.csv](https://github.com/user-attachments/files/18129986/2024-12-13_15-23-25_raw.csv)
[2024-12-13_15-23-25_rowingData.xlsx](https://github.com/user-attachments/files/18129987/2024-12-13_15-23-25_rowingData.xlsx)

How can I fix it ? should I tweak some parameters ?
Thanks for your help

**Comment by JaapvanEkris** [comment]
You could take a look at https://github.com/JaapvanEkris/openrowingmonitor/discussions/58 , which discusses this machine and its settings as well.

Otherwise, take a look at https://github.com/JaapvanEkris/openrowingmonitor/blob/V1Beta-Architecture_Revision/docs/rower_settings.md where we explain the basics for creating/tweaking a config for a rower.
